label_studio/ml_model_providers/models.py

```diff
@@ -0,0 +1,58 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
+
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class ModelProviderConnection(models.Model):
+
+    class ModelProviders(models.TextChoices):
+        OPENAI = 'OpenAI', _('OpenAI')
+
+
+    class ModelProviderConnectionScopes(models.TextChoices):
+        ORG = 'Org', _('Organization')
+        USER = 'User', _('User')
+        MODEL = 'Model', _('Model')
+
+
+    provider = models.CharField(
+        max_length=10,
+        choices=ModelProviders.choices,
+        default=ModelProviders.OPENAI
+    )
+
+    api_key = models.TextField(
+        _('api_key'),
+        null=True,
+        blank=True,
+        help_text='Model provider API key'
+    )
+
+    scope = models.CharField(
+        max_length=10,
```

Comment:
ditto comment above re: `max_length` being overly restrictive (here for example, it would prevent the use of `'Organization'`